VTEX - Ad Network API.json

```diff
@@ -608,6 +617,399 @@
+                            "schema": {
+                                "type": "object",
+                                "description": "Order event request body.",
+                                "required": [
+                                    "accountName",
+                                    "macId",
+                                    "url",
+                                    "ref",
+                                    "sessionId",
+                                    "orderGroup",
+                                    "channel"
+                                ],
+                                "properties": {
+                                    "accountName": {
+                                        "type": "string",
+                                        "description": "Merchant's account name. Should be the same as the account name registered at VTEX.",
+                                        "example": "apiexamples"
+                                    },
+                                    "macId": {
+                                        "type": "string",
+                                        "description": "A long-life identifier stored in the browser cookies or device of each shopper. It is valid for one year and is renewed as new events are sent. It expires if the user clears their cookies or deletes the app. Use UUID Version 4.",
+                                        "example": "f805a89b-ef06-4d6a-ae9d-4dd5699ccd6f"
+                                    },
+                                    "url": {
+                                        "type": "string",
+                                        "description": "Current page URL or the current mobile view.",
+                                        "example": "https://www.apiexamples.com.br/busca?tshirt"
+                                    },
+                                    "ref": {
+                                        "type": "string",
+                                        "description": "Previous page URL (accessed before the current page URL) or mobile equivalent.",
+                                        "example": "https://www.apiexamples.com.br"
+                                    },
+                                    "sessionId": {
+                                        "type": "string",
+                                        "description": "Session identifier, which expires after 30 minutes. Use UUID Version 4.",
+                                        "example": "fb252d90-7494-4097-90fd-cc73eeaa575c"
+                                    },
+                                    "orderGroup": {
+                                        "type": "string",
+                                        "description": "OMS order group for the created order. The orderGroup should be the same as the one created in the OMS to enable the correlation between the order events and the order tables.",
```

Comment:
It is not incorrect, but it could be more clear. There is this example of description, if you wish to incorporate something:

"Order group ID is a segment of the order ID that groups all orders related to the same purchase. For example, in order ID `v71021570str-02`, the order group ID is `v71021570str`.